data/blog/2024-01-31-vector-announcement.mdx

```diff
@@ -0,0 +1,198 @@
+---
+slug: introducing-vector-database
+title: "Upstash Vector: Serverless Vector Database for AI and LLMs"
+authors:
+  - mehmet
+tags: [serverless, database, vector, announcement, ai, llm]
+---
+
+Today, we're introducing Upstash Vector, a serverless vector database built for working with vector embeddings for AI models and LLMs. Upstash Vector will help you to efficiently store and query high-dimensional vector embeddings. Upstash Vector operates on a serverless model, abstracting away hosting and management complexities.
+
+
+### What's a Vector Database really?
+
+A vector database is a specialized database designed to store, manage, and query/search vector embeddings, which are mathematical representations of data points in a high-dimensional space. A vector database supports various distance metrics (such as cosine or euclidean) to measure the similarity between vectors.
+Although storage and managing vector data is straightforward, querying for vector similarity poses significant challenges.
```

Comment:
**Suggested**: Although storing and managing vector data is straightforward, querying for vector similarity poses significant challenges.